### PR TITLE
Ad.jsx: Send auth token for logged in users

### PR DESCRIPTION
**Pull request by dwick**
:eyeglasses: @ajacksified @zeantsoi 

Missing bearer token, this fixes votes and frequency capping for logged in users.

### CODE REVIEW

File: src/config.es6.js [hunk 1]
```diff
@@ -24,7 +24,7 @@ var config = {
 
   loginPath: process.env.LOGIN_PATH || '/oauth2/login',
 
-  adsPath: process.env.ADS_PATH ||  'https://www.reddit.com/api/request_promo.json',
+  adsPath: process.env.ADS_PATH ||  '/api/request_promo.json',
```
Comment (zeantsoi) [inline src/config.es6.js]: Do we know that switching up to a relative path won't have any issues elsewhere in the code, [like here](https://github.com/reddit/reddit-mobile/blob/master/src/views/pages/index.jsx#L187)?

**Comment by zeantsoi** [comment]
One comment, else :fish:

File: src/config.es6.js (hunk 1 above)
Comment (dwick) [inline src/config.es6.js]: that's fine

**Comment by dwick** [comment]
:haircut: @ajacksified trying to get this out asap

**Comment by dwick** [comment]
da8d6477635021c2dcc94910689c1197956d17b6

**Pull request closed without merging**